app/templates/download_framework_users.html

```diff
@@ -24,13 +24,13 @@
       <h1 class="govuk-heading-xl">Download supplier lists for {{ framework['name'] }}</h1>
 
       <div class="explanation-list">
-          <p class="govuk-body">These lists contain details of all suppliers who:</p>
-          <ul class='list-bullet'>
-            <li>started an application</li>
-            <li>are on the framework</li>
-            <li>weren't accepted on the framework</li>
-            <li>have been excluded from the framework</li>
-          </ul>
+        <p class="govuk-body">These lists contain details of all suppliers who:</p>
+        <ul class='govuk-list govuk-list--bullet'>
```

Comment:
Can we change these to double quotes to avoid the problem with searching for them in future?
```suggestion
        <ul class="govuk-list govuk-list--bullet">
```